apps/drec-ui/src/apps/device-group/view/containers/card/MyDeviceGroupCardsList/MyDeviceGroupCardsList.effects.ts

```diff
@@ -0,0 +1,18 @@
+import { DeviceGroupDTO } from '@energyweb/origin-drec-api-client';
+import { useState } from 'react';
+
+export const useMyDeviceGroupCardsListEffects = (deviceGroups: DeviceGroupDTO[]) => {
+    const [selected, setSelected] = useState<DeviceGroupDTO['id']>(null);
```

Comment:
Do we actually need this state here? 
In Origin it was used for showing Request Certificates sidebar / modal, is it the same case here?